src/component/MovieListPage/MovieListPage.js

```diff
@@ -166,7 +238,7 @@ function MovieListPage({ }) {
                     {movieList.map((item) => {
```

Comment:
Can compose this to a separate component

Reply:
Fixed as part of a5b5e33